SPECS/ruby/ruby.spec

```diff
@@ -407,6 +407,9 @@ sudo -u test make test TESTS="-v"
 %{_rpmconfigdir}/rubygems.con
 
 %changelog
+* Tue Sep 10 2024 Harshit Gupta <guptaharshit@microsoft.com> - 3.3.3-2
+- Bump release to build with new rubygem-rexml to fix CVE-2024-41946
```

Comment:
is the rubygem-rexml 3.3.3 compatible with ruby 3.3.3 (we don't need to update this as well)?

Reply:
I think this page clarfies that for rexml 3.3.3 you just need ruby
>= 2.5.0
[rexml | RubyGems.org | your community gem host](https://rubygems.org/gems/rexml/versions/3.3.3)